swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if not inventory:
```

Comment:
Nice job handling the default inventory case. One thing to notice is that you might want to explicitly check for `None` rather than falsy, since written this way, if the caller explicitly passed in `[]` (their own empty list), this logic would still replace the passed in empty list with the new local empty list.